src/controllers/userController.js

```diff
@@ -0,0 +1,156 @@
+import { userServices } from './../services/userService.js';
+import { v4 as uuidv4 } from 'uuid';
+
+let user = '';
+
+const defaultChatId = uuidv4();
+
+const rooms = [
+  {
+    id: defaultChatId,
+    creator: '',
+    name: 'Main-room',
+    history: [],
+  },
+];
+
+function messageValidation(message) {
+  if (!message || message.trim().length === 0) {
+    return 'Message required';
+  }
+
+  if (message.trim().length > 3000) {
+    return 'Message is too long';
+  }
+}
+
+function chatNameValidation(newChatName) {
+  if (!newChatName || newChatName.trim().length === 0) {
+    return 'Chat name required';
+  }
+
+  if (newChatName.trim().length < 2) {
+    return 'Chat name is too short';
+  }
+
+  if (newChatName.length > 30) {
+    return 'Less then 30 characters required';
+  }
+}
```

Comment:
The `chatNameValidation` function does not return anything when the chat name is valid. It would be better to explicitly return `undefined` or `null` at the end of the function to indicate that there are no validation errors.